bootstrapvz/providers/gce/tasks/image.py

```diff
@@ -56,6 +26,26 @@ def run(cls, info):
 		if 'description' in info.manifest.image:
 			image_description = info.manifest.image['description']
 		log_check_call(['gcutil', '--project=' + info.manifest.image['gce_project'],
-		                'addimage', info._gce['image_name'],
-		                info.manifest.image['gcs_destination'] + info._gce['tarball_name'],
+		                'addimage', info.image['name'],
+		                info.manifest.image['gcs_destination'] + info.image['tarball_name'],
 		                '--description=' + image_description])
+
+
+class GenerateImageTarballName(Task):
+	description = 'Generating custom image tarball name'
+	phase = phases.image_registration
+	successors = [image.CreateImageTarball]
+
+	@classmethod
+	def run(cls, info):
+		import datetime
+		today = datetime.datetime.today()
+		name_suffix = today.strftime('%Y%m%d')
+		image_name_format = '{lsb_distribution}-{lsb_release}-{release}-v{name_suffix}'
+		image_name = image_name_format.format(lsb_distribution=info._gce['lsb_distribution'],
```

Comment:
It would be cool if we could pre-computer the `lsb*` vars and add them to the manifest vars. On the other hand: https://en.wikipedia.org/wiki/Linux_Standard_Base#Limitations_on_Debian